code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1285,17 +1285,16 @@
 		dna.species.create_organs(src)
 
 		//Apply relevant damages and variables to the new organs.
-		for(var/B in bodyparts)
-			var/obj/item/organ/external/E = B
+		for(var/obj/item/organ/external/E in bodyparts)
```

Comment:
```suggestion
		for(var/obj/item/organ/external/E as anything in bodyparts)
```
can be made to not filter as all `bodyparts` are always of this type.
Reject typeless. `as anything` is the new black